ArduPlane/Plane.h

```diff
@@ -252,6 +253,8 @@ class Plane : public AP_HAL::HAL::Callbacks {
 
     // GCS selection
     AP_SerialManager serial_manager;
+    const uint8_t num_gcs = MAVLINK_COMM_NUM_BUFFERS;
```

Comment:
Is this just a rebase update issue?  I don't think this has anything to do with the soaring code.